aeneas/src/ssa/SsaInterpreter.v3

```diff
@@ -2,13 +2,11 @@
 // See LICENSE for details of Apache 2.0 license.
 
 def INITIAL_VALUES = 128;
-def INITIAL_FRAMES = 16;
 
 // Implements an interpreter capable of executing all SSA instructions, calls, etc
 // for initialization and testing.
 class SsaInterpreter(prog: Program, genSsa: (IrSpec, int) -> SsaGraph) {
 	var values = Array<Val>.new(INITIAL_VALUES);
-	var stack = Array<SsaIntFrame>.new(INITIAL_FRAMES);
 	var frame: SsaIntFrame;
 	def args = SsaIntArguments.new(prog);
 	var sp: int;
```

Comment:
Is it possible to remove `sp` as well?

Reply:
`traceInstr()` uses `sp` to decide indent

Reply:
What about just walking the frames? It's already linear in the number of frames, since it counts from `0` to `sp`.